packages/search/package.json

```diff
@@ -21,11 +21,18 @@
     "test": "export COVERAGE=true; ember test --test-port 0"
   },
   "dependencies": {
+    "ember-ajax": "^5.0.0",
```

Comment:
What do we need ajax for? I thought we were trying to move away from ajax and towards the fetch api.